app.yaml

```diff
@@ -80,6 +80,11 @@ handlers:
   secure: always
   http_headers:
     Cache-Control: 'no-cache'
+# To serve extension resources in dev mode.
+- url: /extensions/(gadgets|interactions|rich_text_components)/(.*\.(js))$
+  static_files: extensions/\1/\2
```

Comment:
Yes, they are inserting the capture groups from the pattern. I'll update the comment.